pyleecan/Methods/Mesh/MeshVTK/convert.py

```diff
@@ -34,73 +39,69 @@ def convert(self, meshtype, scale):
 
     if meshtype == "MeshVTK":
         new_mesh = self.copy()
+        new_mesh.node.coordinate *= scale
     elif meshtype == "MeshMat":
+        from ....Classes.MeshMat import MeshMat
```

Comment:
The objective was to avoid circular import with MeshMat importing MeshVTK and MeshVTK importing MeshMat.
Is there a standard way to solve such issue ?